app/lib/features/onboarding/pages/login_page.dart

```diff
@@ -54,7 +54,9 @@ class _LoginPageState extends ConsumerState<LoginPage> {
           // no message means, login was successful.
           context.goNamed(Routes.main.name);
         } else {
-          customMsgSnackbar(context, loginSuccess);
+          EasyLoading.showError(
```

Comment:
could use ```showSuccess``` here.

Reply:
no... if we have a message it is an "error message" ... this is a bit confusing in the API but that's how it is ...